src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -218,12 +392,16 @@ public class CRLP_RollupAccount_TEST {
         UTIL_CurrencyCache.instance = mockCurrencyCache;
 
         // Define a simple 2x exchange rate to use for this test
-        mockCurrencyCache.effectiveRatesReturn = new List<Decimal>{ 2.0 };
-        mockCurrencyCache.effectiveDatesReturn = new List<Date>{ Date.newInstance(1900,1,1) };
+        mockCurrencyCache.effectiveRatesReturn = new List<Decimal>{
+            2.0
+        };
+        mockCurrencyCache.effectiveDatesReturn = new List<Date>{
+            Date.newInstance(1900,1,1)
+        };
 
         // Test is only relevant if the multicurrency org actually has multiple currencies.   If not, skip it
         SObject[] currencies = Database.query('SELECT IsoCode FROM CurrencyType WHERE IsActive = True');
-        if (currencies.size() <= 1) {
+        if (currencies.isEmpty()) {
```

Comment:
This was returning when there was 1 or less before, was this change to empty only intentional?

Reply:
I reset the class to its original state.